xsgen/openmc_origen.py

```diff
@@ -151,15 +152,16 @@ def __init__(self, rc):
         self.eafds = data_source.EAFDataSource()
         self.omcds = data_source.OpenMCDataSource(
                         cross_sections=rc.openmc_cross_sections,
-                        src_group_struct=np.logspace(1, -9, 1001))
+                        src_group_struct=rc.openmc_group_struct)
         data_sources = [self.omcds]
-        if not rc.is_thermal:
-            data_sources.append(self.eafds)
+        #if not rc.is_thermal:
+        #    data_sources.append(self.eafds)
```

Comment:
I didn't want to fully remove it. I just wanted to get it to the point where thermal reactors were working.